02_configure_host.sh

```diff
@@ -119,12 +119,12 @@ EOF
 
```

Comment:
On L106, that `if [ "$PRO_IF" ]` needs to be fixed. It is running the contents of `$PRO_IF` variable...

Also, curlys missing all over.